docs/api/textarea.md

```diff
@@ -29,6 +29,40 @@ import BasicPlayground from '@site/static/usage/v7/textarea/basic/index.md';
 
 <BasicPlayground />
 
+## Label Positioning
+
+Labels will take up the width of their content by default. This positioning can be changed to be a fixed width, stacked, or floating label.
+
+import Labels from '@site/static/usage/v7/textarea/labels/index.md';
+
+<Labels />
+
+## Filled Textareas
+
+Material Design offers filled styles for a textarea. The `fill` property on the item can be set to either `"solid"` or `"outline"`.
+
+Since the `fill` styles visually defines the textarea container, textareas that use `fill` should not be used in `ion-item`.
+
+import Fill from '@site/static/usage/v7/textarea/fill/index.md';
+
+<Fill />
+
+## Helper & Error Text
+
+Helper and error text can be used inside of a textarea with the `helperText` and `errorText` property. The error text will not be displayed unless the `ion-invalid` class is added to the `ion-textarea`. In Angular, this is done automatically through form validation. In JavaScript, React and Vue, the class needs to be manually added based on your own validation.
+
+import HelperError from '@site/static/usage/v7/textarea/helper-error/index.md';
+
+<HelperError />
+
+## Textarea Counter
+
+The textarea counter is text that displays under a textarea to notify the user of how many characters have been entered out of the total that the textarea will accept. When adding counter, the default behavior is to format the value that gets displayed as `inputLength` / `maxLength`. This behavior can be customized by passing in a formatter function to the `counterFormatter` property.
+
+import Counter from '@site/static/usage/v7/textarea/counter/index.md';
+
+<Counter />
```

Comment:
I don't see the bottom content when opening the React Stackblitz here either; I wonder if that's just an Ionic bug?